### PR TITLE
Completed Doctor CRUD

### PR DESCRIPTION
**Pull request by aakash1692**
This PR holds the code for Doctor CRUD operations. Changes:
- _Department_ field as optional
- Modified fastify Route Doctor schema, in this also declared _"Department"_ field as optional
- GetDoctors function capable to query data using query strings

### CODE REVIEW

**Comment by aakash1692** [comment]
Guys please review my PR holding completed changes for Doctors CRUD

**Review by ankitchawla777 — CHANGES_REQUESTED** [review]
I think we need to export getDoctorByQuery

File: doctor/doctors.js [hunk 1]
```diff
@@ -2,25 +2,82 @@ const { Doctor } = require("./doctor");
...
+    let response = await Doctor.getDoctors();
+    
+    res
+      .code(response.statusCode)
+      .send(response);
   } catch (err) {}
 };
 
-exports.getDoctorByEmail = async (req, res) => {
+const getDoctorByQuery = async (req, res) => {
```
Comment (ankitchawla777) [inline doctor/doctors.js:17]: export this

Comment (aakash1692) [reply doctor/doctors.js:17]: Actually no need to:
- first, you cant have two endpoints with same structure.
- second, you can hit getDoctors either with query-strings to get data based on your queries or, without query-strings to get all doctors.

**Pull request merged**